scripts/chain-prep.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#set -eux
+killall node
+ganache-cli -u 0 -u 1 -u 2 -m 'cook mango twist then skin sort option civil have still rather guilt' &
+git clone https://github.com/althea-mesh/simple-bidirectional-erc20-channel contract
+pushd contract 
+npm install .
+truffle compile
+addresses="$(truffle migrate --verbose)"
+export CHANNEL_ADDRESS="$(echo $addresses | grep -Eo ': 0x[0-9a-f]{40}'| cut -d$'\n' -f3)"
```

Comment:
I think I found out a reliable way to obtain the address with this comand:

```sh
truffle migrate --verbose
export CHANNEL_ADDRES=`jq -r '.networks| to_entries | sort_by(.key) | last.value.address' build/contracts/ChannelManager.json`
```
(as default network id in ganache is unix timestamp, so this command will get the address from the largest network in the JSON)

Also on a side note wouldn't it be good idea to remove `build/` dir from the contract and add it to .gitignore? It changes after every build with `truffle compile`. Wouldn't it be also a security concern if we'd make the contract address from the mainnet committed to the repo?

Reply:
what makes this way superior? I don't really care how we find the address all that much. 

adding build to gitigore is a good idea. 

How could the contract address be a problem? Everyone needs to know it to interact with it.